scripts/utils/build.js

```diff
@@ -89,3 +114,39 @@ export async function rollupScript (scriptPath, name, supportsMozProxies = false
     const generated = await bundle.generate(outputOptions)
     return generated.output[0].code
 }
+
+/**
+ * Decide when to stub a module, or include it.
+ *
+ * This plugin watches every import, when it sees a top-level feature file
+ * it checks the platformSupport to see if the feature should be included
+ *
+ * @param {string} platform
+ */
+function stubFeatures (platform) {
+    const dir = join(CWD, 'src/features')
+    const featureNames = platformSupport[platform]
+    return {
+        name: 'platformFeatures',
+        load (id) {
+            if (!id.startsWith(dir)) return null
+
+            // grab the first path segment
+            const base = id.slice(dir.length + 1).split('/')[0]
+
+            // if it's a directory, skip
+            if (!base.endsWith('.js')) return null
+
+            // convert the pathname to a feature name
+            const featureName = base.replace(/-(.)/g, (match, letter) => {
+                return letter.toUpperCase()
+            }).slice(0, -3)
+
+            // if the feature is listed for this platform, include it
+            if (featureNames.includes(featureName)) return null
+
+            // if we get here the feature should be skipped/stubbed
+            return 'export default ' + JSON.stringify('__stub__' + base)
```

Comment:
this is all a bit of a hack, but I had the idea of how to prevent Duck Player bloating every bundle by simply 'skipping' some modules in a low-risk-easy-to-ship way. 

as always fully open to other ideas 🙏🏻

Reply:
I wonder if we should just fork the import module instead, that's why it's always included and not tree shaked away.

Reply:
Looks like they support 'include' now, can we check if that works?